test/test_util.py

```diff
@@ -115,7 +115,7 @@ def popen_fail(*args, **kwargs):
         with self.assertRaises(subprocess.CalledProcessError) as exc_context:
             util.command_output([b"taga", b"\xc3\xa9"])
         self.assertEqual(exc_context.exception.returncode, 1)
-        self.assertEqual(exc_context.exception.cmd, b"taga \xc3\xa9")
+        self.assertEqual(exc_context.exception.cmd, 'taga \xc3\xa9')
```

Comment:
Right; that makes sense—because on Unixes, the command is now a surrogate-escaped Unicode string, and on Windows, commands are just plain Unicode. Such are the hazards of trying to pretend arguments are Unicode on Unix. :cry: